src/pe-flyout-menu-categories/widget/pe-flyout-menu-categories.u-option-schema.json

```diff
@@ -0,0 +1 @@
+[{"displayValue":"Category","name":"categories","display_value_list":[],"section":"other","label":"categories","type":"glide_list","value":"sc_category","ed":{"reference":"sc_category"}}]
```

Comment:
Please format this JSON so that it renders well on the library portal and reads well.